SpatialGDK/Extras/fastbuild/uninstall.ps1

```diff
@@ -36,14 +36,19 @@ if ($service) {
     Stop-Process -Name "FBuildWorker.exe.copy" -ErrorAction SilentlyContinue
 }
 
+# Manually remove marker file from the brokerage, since it's not possible to cleanly exit FBuildWorker.exe
+$brokerageFile = [System.IO.Path]::Combine($env:FASTBUILD_BROKERAGE_PATH, "main", "17", [System.Net.Dns]::GetHostName())
+if (Test-Path $brokerageFile) {
+    Write-Host "Removing from brokerage..."
+    [System.IO.File]::Delete($brokerageFile)
+}
+
 Write-Host "Cleaning environment..."
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_EXE_PATH", $null, [System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_CACHE_PATH", $null, [System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_BROKERAGE_PATH", $null, [System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable("FASTBUILD_CACHE_MODE", $null, [System.EnvironmentVariableTarget]::Machine)
 
-refreshenv
```

Comment:
Why is refreshenv not needed anymore?

Reply:
It never worked, and it's unlikely anyone will be trying to run fbuild commands from that powershell prompt.